chapters/annotations.tex

```diff
@@ -68,27 +73,190 @@ \section{Annotations for Documentation}\doublelabel{annotations-for-documentatio
 \begin{lstlisting}[language=modelica]
 <a href="Modelica://MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
-\begin{lstlisting}[language=grammar]
-preferred-view-annotation:
-   annotation "(" preferredView "=" ("info" | "diagram" | "text") ")"
+
+\subsection{Annotations for Figures}\doublelabel{annotations-for-figures}
+The \lstinline!figure-annotations! is a list of \lstinline!Figure! annotations given by:
+\begin{lstlisting}[language=modelica]
+record Figure
+  String title = "" "Title meant for display";
+  String identifier "Identifier meant for programmatic access";
+  String group = "" "Name of plot group";
+  Boolean preferred = false "Automatically display figure after simulation";
+  Plot[:] plots "Plots";
+  String caption "Figure caption";
+end Figure;
+\end{lstlisting}
+
+A \lstinline!Figure! is a graphical container that can contain several
+\lstinline!plots! described by \lstinline!Plot! annotations:
+\begin{lstlisting}[language=modelica]
+record Plot
+  String title "Title meant for display";
+  String identifier "Identifier meant for programmatic access";
+  Curve[:] curves "Plot curves";
+  Axis x "X axis properties";
+  Axis y "Y axis properties";
+end Plot;
+\end{lstlisting}
+
+A \lstinline!Plot! can contain several \lstinline!curves!, see
+\autoref{plot-curves}, that all share a common x and y axis with properties
+described in \autoref{axis-properties}.
+
+Both \lstinline!Figure! and \lstinline!Plot! can have an optional title. When
+the \lstinline!Figure! \lstinline!title! is the empty string (the default), the
+tool must produce a non-empty title based on the figure content.  On the other
+hand, the \lstinline!Plot! \lstinline!title! has a tool-dependent default, but
+the default may be the empty string.  When the Plot title is the empty string, no
+title should be shown. The plot title is not to be confused with the plot
```

Comment:
I think one has to choose between "`Plot` `title`" and "plot title".  It makes sense to say "plot title" when relating to "plot label", but I think the "Plot title" should be "`Plot` `title`".